### PR TITLE
News for OCaml 4.14.1 first release candidate

### PR DESCRIPTION
**Pull request by Octachron**
This PR adds the announce for the first release candidate of OCaml 4.14.1 (hopefully updated to ocaml.org format).

### CODE REVIEW

File: data/news/platform/ocaml-4.14.1.rc1.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+---
+title: OCaml 4.14.1 - First release candidate
```
Comment (christinerose) [inline data/news/platform/ocaml-4.14.1.rc1.md]:
```suggestion
title: OCaml 4.14.1 - First Release Candidate
```
Also, perhaps just “Release Candidate,” as David said there probably won’t be another one.

File: data/news/platform/ocaml-4.14.1.rc1.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+In both cases, all available options can be listed with `opam search ocaml-option`.
+
+---
+
+## Changes Since OCaml 4.14.0
+
+
+### Compiler User-Interface and Warnings:
+
+- [#11184](https://github.com/ocaml/ocaml/issues/11184), [#11670](https://github.com/ocaml/ocaml/issues/11670): Stop calling ranlib on created / installed libraries
```
Comment (christinerose) [inline data/news/platform/ocaml-4.14.1.rc1.md]:
```suggestion
- [#11184](https://github.com/ocaml/ocaml/issues/11184), [#11670](https://github.com/ocaml/ocaml/issues/11670): Stop calling `ranlib` on created / installed libraries
```

File: data/news/platform/ocaml-4.14.1.rc1.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+  (Sébastien Hinderer and Xavier Leroy, review by the same)
+
+### Build System:
+
+- [#11370](https://github.com/ocaml/ocaml/issues/11370), [#11373](https://github.com/ocaml/ocaml/issues/11373): Don't pass CFLAGS to flexlink during configure.
+  (David Allsopp, report by William Hu, review by Xavier Leroy and
+   Sébastien Hinderer)
+
+- [#11487](https://github.com/ocaml/ocaml/issues/11487): Thwart FMA test optimization during configure
```
Comment (christinerose) [inline data/news/platform/ocaml-4.14.1.rc1.md]:
```suggestion
- [#11487](https://github.com/ocaml/ocaml/issues/11487): Thwart FMA test optimisation during configure
```
The Tarides Style Guide goes with the UK spelling, but I’m not sure that applies here.

File: data/news/platform/ocaml-4.14.1.rc1.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+  (Jacques Garrigue, report by François Thiré, review by Matthew Ryan)
+
+- [#11204](https://github.com/ocaml/ocaml/issues/11204): Fix regression introduced in 4.14.0 that would trigger Warning 17 when
+  calling virtual methods introduced by constraining the self type from within
+  the class definition.
+  (Nicolás Ojeda Bär, review by Leo White)
+
+- [#11263](https://github.com/ocaml/ocaml/issues/11263), [#11267](https://github.com/ocaml/ocaml/issues/11267): caml/{memory,misc}.h: check whether `_MSC_VER` is defined
+  before using it to ensure that the headers can always be used in code which
+  turns on -Wundef (or equivalent).
```
Comment (christinerose) [inline data/news/platform/ocaml-4.14.1.rc1.md]:
```suggestion
  turns on `-Wundef` (or equivalent).
```

File: data/news/platform/ocaml-4.14.1.rc1.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+  (David Allsopp and Nicolás Ojeda Bär, review by Nicolás Ojeda Bär and
+   Sébastien Hinderer)
+
+- [#11314](https://github.com/ocaml/ocaml/issues/11314), [#11416](https://github.com/ocaml/ocaml/issues/11416): fix non-informative error message for module inclusion
+  (Florian Angeletti, report by Thierry Martinez, review by Gabriel Scherer)
+
+- [#11358](https://github.com/ocaml/ocaml/issues/11358), [#11379](https://github.com/ocaml/ocaml/issues/11379): Refactor the initialization of bytecode threading,
```
Comment (christinerose) [inline data/news/platform/ocaml-4.14.1.rc1.md]:
```suggestion
- [#11358](https://github.com/ocaml/ocaml/issues/11358), [#11379](https://github.com/ocaml/ocaml/issues/11379): Refactor the initialisation of bytecode threading,
```
Same UK spelling comment as above, if applicable.

File: data/news/platform/ocaml-4.14.1.rc1.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+   Gabriel Scherer)
+
+- [#11622](https://github.com/ocaml/ocaml/issues/11622): Prevent stack overflow when printing a constructor or record
+  mismatch error involving recursive types.
+  (Florian Angeletti, review by Gabriel Scherer)
+
+- [#11732](https://github.com/ocaml/ocaml/issues/11732): Ensure that types from packed modules are always generalised
+  (Stephen Dolan and Leo White, review by Jacques Garrigue)
+
+- [#11737](https://github.com/ocaml/ocaml/issues/11737): Fix segfault condition in Unix.stat under Windows in the presence of
```
Comment (christinerose) [inline data/news/platform/ocaml-4.14.1.rc1.md]:
```suggestion
- [#11737](https://github.com/ocaml/ocaml/issues/11737): Fix segfault condition in `Unix.stat` under Windows in the presence of
```

File: data/news/platform/ocaml-4.14.1.rc1.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+  multiple threads.
+  (Marc Lasson, Nicolás Ojeda Bär, review by Gabriel Scherer and David Allsopp)
+
+- [#11776](https://github.com/ocaml/ocaml/issues/11776): Extend environment with functor parameters in `strengthen_lazy`.
+  (Chris Casinghino and Luke Maurer, review by Gabriel Scherer)
+
+- [#11533](https://github.com/ocaml/ocaml/issues/11533), [#11534](https://github.com/ocaml/ocaml/issues/11534): follow synonyms again in #show_module_type
```
Comment (christinerose) [inline data/news/platform/ocaml-4.14.1.rc1.md]:
```suggestion
- [#11533](https://github.com/ocaml/ocaml/issues/11533), [#11534](https://github.com/ocaml/ocaml/issues/11534): follow synonyms again in `#show_module_type`
```

File: data/news/platform/ocaml-4.14.1.rc1.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+  (this had stopped working in 4.14.0)
+  (Gabriel Scherer, review by Jacques Garrigue, report by Yaron Minsky)
+
+- [#11768](https://github.com/ocaml/ocaml/issues/11768), [#11788](https://github.com/ocaml/ocaml/issues/11788): Fix crash at start-up of bytecode programs in
+  no-naked-pointers mode caused by wrong initialization of caml_global_data
```
Comment (christinerose) [inline data/news/platform/ocaml-4.14.1.rc1.md]:
```suggestion
  no-naked-pointers mode caused by wrong initialisation of `caml_global_data`
```
UK spelling, unless American spelling used on ocaml.org. Let me know, and I’ll update the style guide for this. (American spelling is used for Dune docs, for example.)

**Review by christinerose — CHANGES_REQUESTED** [review]
Just some minor formatting changes with questions about UK vs. US spelling.

File: data/news/platform/ocaml-4.14.1.rc1.md (hunk 1 above)
Comment (Octachron) [reply data/news/platform/ocaml-4.14.1.rc1.md]: Good point.

File: data/news/platform/ocaml-4.14.1.rc1.md (hunk 8 above)
Comment (Octachron) [reply data/news/platform/ocaml-4.14.1.rc1.md]: That's a good question, this part is an extract from the official change log. There is value in keeping exactly the same entries but at the same time some level of copy editing sounds fine.

**Comment by tmattio** [comment]
Re. UK vs US spelling, I don't have a strong opinion, but:
- I remember reading that UK spelling was more widespread, so it might be better in terms of inclusivity
- We used UK spelling for other content, including the Why OCaml page, documentation, etc.

So unless there are arguments in favour of US spelling, let's go with UK (as you just did in the PR, thank you!)

**Review by tmattio — APPROVED** [review]

File: data/news/platform/ocaml-4.14.1.rc1.md (hunk 1 above)
Comment (christinerose) [reply data/news/platform/ocaml-4.14.1.rc1.md]: It's best if this is capitalised, as that's correct in title case

**Pull request merged**